getting-started/templates/systemlink-values.yaml

```diff
@@ -334,6 +334,31 @@ dashboardhost:
     extraPaths: []
     tls: []
 
+  ## This API ingress makes it possible to expose elements of the Grafana backend API via the API ingress.
+  ##
+  apiIngress:
+    ## Enable the ingress.
+    ##
+    enabled: true
+    ## For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
+    ## See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#specifying-the-class-of-an-ingress
+    # <ATTENTION> - Configure you ingress controller class name.
+    ##
+    ingressClassName: "nginx"
```

Comment:
We don't ask users to do this for any other container. Is Grafana special for some reason?

Reply:
Talking with the team, it doesn't sound like we had a good reason for this. We're setting it to an empty string in our default values file. I'm leaning toward removing it from both grafana ingresses here.

Reply:
Currently nginx is the only ingress controller we officially support for customer deployments. In the future we should either have a global configuration for common ingress settings or switch to a service mesh or other standard system that we control. I hope we don't ever require users to individually configure each ingress, as that would be a maintenance nightmare.

Reply:
Yeah, that makes sense.

Reply:
I removed it from both ingresses for Grafana.

Reply:
So, I guess my question comes back to "Is this PR necessary?". If these are all default values we don't need to duplicate them here.

Reply:
Ah, I thought your question was scoped to the `ingressClassName` property and whether that was necessary.

This PR adds the new API ingress for grafana to this values file for the purpose of documentation, if nothing else. We do need both `ingress` and the new `apiIngress` configured for Grafana, with the `hosts` set to the API hosts for the latter. We have "hosts" set to example.com in the default values.

Reply:
Gotcha, I wasn't thinking about the fact that Grafana doesn't automatically uptake the global host configuration. 

I do recommend stripping out any default values we don't expect the user to change here. Like the paths. We should try and keep this file as lean as possible to minimize cognitive load on the user.